templates/managed/kafka.html

```diff
@@ -36,265 +34,600 @@ <h1>
-        <li class="p-list__item is-ticked">Load balancing and data replication allow failure without downtime</li>
-        <li class="p-list__item is-ticked">High availability and throughput through Kafka&rsquo;s distributed cluster architecture</li>
-        <li class="p-list__item is-ticked">Support in deploying your organisation&rsquo;s Schema Registry</li>
+        <li class="p-list__item is-ticked">99.99% SLA, high availability by default</li>
+        <li class="p-list__item is-ticked">Load balancing and data replication</li>
+        <li class="p-list__item is-ticked">Support in deploying Schema Registry</li>
       </ul>
     </div>
-    <div class="col-6 p-card">
-      <h4 class="p-card__title">Tailored to your specific needs</h4>
-      <hr class="u-sv1">
+    <div class="col-6">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/0cfe6c2d-build.svg",
+          alt="VMWare",
+          width="152",
+          height="100",
+          hi_def=True,
+          attrs={"style":"height: 80px; width: auto;"},
+        ) | safe
+      }}
+      <h4>Optimised</h4>
       <ul class="p-list">
-        <li class="p-list__item is-ticked">Control the version of Kafka your cloud runs</li>
+        <li class="p-list__item is-ticked">Kafka experts provide 24/7 management</li>
         <li class="p-list__item is-ticked">Schedule updates to downtime windows</li>
-        <li class="p-list__item is-ticked">Understand key metrics with an integrated  web-based dashboard and logging, monitoring and alerting (Graylog-Prometheus-Grafana) stack</li>
+        <li class="p-list__item is-ticked">Web-based dashboard of key health metrics</li>
       </ul>
     </div>
   </div>
 </section>
 
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-5 u-vertically-center">
+      <div>
+        <h3>Kafka can be challenging. Learn&nbsp;how we simplify it.</h3>
+        <p><a href="https://www.brighttalk.com/webcast/6793/410402" class="js-invoke-modal p-link--external">Watch the full webinar on demand now</a></p>
+      </div>
+    </div>
+    <div class="col-5 col-start-large-7">
+      <a href="https://www.brighttalk.com/webcast/6793/410402" aria-hidden="true" tabindex="-1">
+        {{
+          image(
+            url="https://assets.ubuntu.com/v1/8b5358eb-Banner-Kafka-in-Prod-Webinar.png",
+            alt="",
+            width="1008",
+            height="528",
+            hi_def=True,
+          ) | safe
+      }}
+      </a>
+    </div>
+  </div>
+</section>
+
 <section class="p-strip">
   <div class="row">
-    <div class="col-7">
-      <h2>
-        Kafka management, architecture and operation
-      </h2>
-      <p>Canonical offers a <a href="/managed">fully managed service</a> for open source software and applications, including support for Apache Kafka, anywhere you can run Ubuntu.</p>
+    <div class="col-8">
+      <h2>Kafka on any kubernetes, any cloud and at any scale</h2>
```

Comment:
I think this would look better p-heading--three or a br before the 'and'

Reply:
Done.